packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -340,10 +351,12 @@ export const addAssetToVault = async (
     [issuerName],
   );
 
-  const oracleInstanceName = instanceNameFor(oracleBrand, 'USD');
   // don't add the collateral offering to vaultFactory until its price feed is available
   // eslint-disable-next-line no-restricted-syntax -- allow this computed property
-  await consumeInstance[oracleInstanceName];
+  await consumeInstance[oracleBrandFeedName(oracleBrand, 'USD')];
+  // await also the real brand
```

Comment:
suggestion: editorial:

```suggestion
  // await also the negotiable brand
```